-- DATA CONTAINER --
local function toAnimationString(s) return "rbxassetid://" .. s end
local Data = {
	["67"] = {
		DisplayName = "67",
		AttackAnimation = toAnimationString(0),
		WalkingAnimation = toAnimationString(0),
		AttackDmg = 15,
		Health = 100,
		Rarity = "Common",
		SpawnPrice = 0,
		EnergyCost = 10,
	},
	BallerinaCappuccina = {
		DisplayName = "Ballerina Cappuccina",
		AttackAnimation = toAnimationString(0),
		WalkingAnimation = toAnimationString(0),
		AttackDmg = 15,
		Health = 100,
		Rarity = "Common",
		SpawnPrice = 0,
		EnergyCost = 10,
	},
	BombardiroCrocodilo = {
		DisplayName = "Bombardiro Crocodilo",
		AttackAnimation = toAnimationString(0),
		WalkingAnimation = toAnimationString(0),
		AttackDmg = 15,
		Health = 100,
		Rarity = "Common",
		SpawnPrice = 0,
		EnergyCost = 10,
	},
	BrrBrrPapatim = {
		DisplayName = "Brr Brr Papatim",
		AttackAnimation = toAnimationString(0),
		WalkingAnimation = toAnimationString(0),
		AttackDmg = 15,
		Health = 100,
		Rarity = "Common",
		SpawnPrice = 0,
		EnergyCost = 10,
	},
	CanneloniDragon = {
		DisplayName = "Canneloni Dragon",
		AttackAnimation = toAnimationString(0),
		WalkingAnimation = toAnimationString(0),
		AttackDmg = 15,
		Health = 100,
		Rarity = "Common",
		SpawnPrice = 0,
		EnergyCost = 10,
	},
	CappuccinoAssassino = {
		DisplayName = "Cappuccino Assassino",
		AttackAnimation = toAnimationString(0),
		WalkingAnimation = toAnimationString(0),
		AttackDmg = 15,
		Health = 100,
		Rarity = "Common",
		SpawnPrice = 0,
		EnergyCost = 10,
	},
	LaGrandeCombination = {
		DisplayName = "La Grande Combination",
		AttackAnimation = toAnimationString(0),
		WalkingAnimation = toAnimationString(0),
		AttackDmg = 15,
		Health = 100,
		Rarity = "Common",
		SpawnPrice = 0,
		EnergyCost = 10,
	},
	LiriliLarila = {
		DisplayName = "Lirili Larila",
		AttackAnimation = toAnimationString(0),
		WalkingAnimation = toAnimationString(0),
		AttackDmg = 15,
		Health = 100,
		Rarity = "Common",
		SpawnPrice = 0,
		EnergyCost = 10,
	},
	NoobiniPizzanini = {
		DisplayName = "Noobini Pizzanini",
		AttackAnimation = toAnimationString(0),
		WalkingAnimation = toAnimationString(0),
		AttackDmg = 15,
		Health = 100,
		Rarity = "Common",
		SpawnPrice = 0,
		EnergyCost = 10,
	},
	OdinDinDinDun = {
		DisplayName = "Odin Din Din Dun",
		AttackAnimation = toAnimationString(0),
		WalkingAnimation = toAnimationString(0),
		AttackDmg = 15,
		Health = 100,
		Rarity = "Common",
		SpawnPrice = 0,
		EnergyCost = 10,
	},
	OrcaleroOrcala = {
		DisplayName = "Orcalero Orcala",
		AttackAnimation = toAnimationString(0),
		WalkingAnimation = toAnimationString(0),
		AttackDmg = 15,
		Health = 100,
		Rarity = "Common",
		SpawnPrice = 0,
		EnergyCost = 10,
	},
	ToToToSahur = {
		DisplayName = "To To To Sahur",
		AttackAnimation = toAnimationString(0),
		WalkingAnimation = toAnimationString(0),
		AttackDmg = 15,
		Health = 100,
		Rarity = "Common",
		SpawnPrice = 0,
		EnergyCost = 10,
	},
	TralaleroTralala = {
		DisplayName = "Tralalero Tralala",
		AttackAnimation = toAnimationString(0),
		WalkingAnimation = toAnimationString(0),
		AttackDmg = 15,
		Health = 100,
		Rarity = "Common",
		SpawnPrice = 0,
		EnergyCost = 10,
	},
	TrippiTroppi = {
		DisplayName = "Trippi Troppi",
		AttackAnimation = toAnimationString(0),
		WalkingAnimation = toAnimationString(0),
		AttackDmg = 15,
		Health = 100,
		Rarity = "Common",
		SpawnPrice = 0,
		EnergyCost = 10,
	}
}

export type Brainrot = {
	DisplayName: string,
	AttackAnimation: string,
	WalkingAnimation: string,
	AttackDmg: number,
	Health: number,
	Rarity: string,
	SpawnPrice: number,
	EnergyCost: number,
}

export type Data = typeof(Data)

return Data